ckanext/aircan/logic/action.py

```
@@ -0,0 +1,179 @@
+        client = AirflowClient()
+        try:
+            dag_run = client.get_dag_run(dag_run_id)
+            task_status.update(
+                {
+                    "dag": dag_run,
+                }
+            )
+        except requests.HTTPError as e:
+            log.error(
+                tk._("Failed to fetch Airflow DAG run '%s': %s"), dag_run_id, str(e)
+            )
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)